task_manager/api.py

```diff
@@ -7,11 +7,8 @@
 # from flask import request
 
 from flask_restx import Resource, Api
-
-# from task_manager.routes.tasks import task as task_namespace
-# from task_manager.routes.users import user as user_namespace
-
-from routes.tasks import model_get_create_task, model_post_create_task, model_fetch_task, model_get_update_task, model_post_update_task, model_delete_task
+from routes.tasks import *
```

Comment:
F403: 'from routes.tasks import *' used; unable to detect undefined names  <!-- Automatically posted by Lintly -->